tests-e2e/cypress/integration/frontstage/rhs/latest_update_spec.js

```diff
@@ -130,6 +130,135 @@ describe('incident rhs > latest update', () => {
         });
     });
 
+    describe('the default reminder', () => {
+        it('shows \'none\' when we have not made a previous update', () => {
+            // # Run the /incident status slash command.
```

Comment:
Nit: 
```suggestion
            // # Run the /incident update slash command.
```

The typo is also in the following tests.